deployment/packagebuild/packages.d/vnet/openvnet.spec

```diff
@@ -133,19 +155,35 @@ This package contains OpenVNet's Restful WebAPI. Users can interact with OpenVNe
 %files webapi
 /opt/axsh/openvnet/vnet/rack
 %config(noreplace) /etc/openvnet/webapi.conf
+%if 0%{?el6}
 %config(noreplace) /etc/default/vnet-webapi
 %config /etc/init/vnet-webapi.conf
+%else
+%config(noreplace) /etc/sysconfig/vnet-webapi
+%config %{_unitdir}/vnet-webapi.service
+%config /etc/firewalld/services/vnet-webapi.xml
+%endif
```

Comment:
I see the file `vnet/bin/vnwebapi` being added in this pull request but it is not being installed here. Why not?